avaframe/out3Plot/plotUtilsCfg.ini

```diff
@@ -45,6 +45,9 @@ centile2 = 0.99
 # number of bins between interval defined by centile2
 steps2Centile2 = 50
 
+# levels for discrete colormap if levels and colors NOT provided
+defLevels = 6
```

Comment:
Maybe name this a bit more obvious, like e.g. discreteLevels, numDiscrLevel or similar?

Reply:
changed to discreteLevel